15/stretch-slim/Dockerfile

```diff
@@ -0,0 +1,96 @@
+FROM debian:stretch-slim
+
+RUN groupadd --gid 1000 node \
+  && useradd --uid 1000 --gid node --shell /bin/bash --create-home node
+
+ENV NODE_VERSION 15.0.0
+
+RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
+    && case "${dpkgArch##*-}" in \
+      amd64) ARCH='x64';; \
+      ppc64el) ARCH='ppc64le';; \
+      s390x) ARCH='s390x';; \
```

Comment:
Don't think it's worth trying to work around removing these in the template. I think stripping them from the `architectures` is good enough

Reply:
huh? you want me to remove `s390x`?

Reply:
No, I'm just pointing out that we still have it embed here, but since you removed it from the `architectures` at least it won't get built/tagged

Reply:
it's 1 AM here, so I'm heading to bed. Feel free to make any changes you want and land it, I just want v15 out 🙂

Reply:
> No, I'm just pointing out that we still have it embed here, but since you removed it from the `architectures` at least it won't get built/tagged

I've just copied 14 and run `update`